docs/datamodels.md

```diff
@@ -84,5 +84,9 @@ And here is the same example with Neurobagel annotations added:
 }
 ```
 
-### Special attributes
+Note that we use a Neurobagel namespace (URI: http://neurobagel.org/vocab/) for controlled terms representing classes or properties that we model, such as `"Age"` and `"Sex"`, but that these can have equivalent terms in another namespace we are using. For example, the following terms from the Neurobagel annotations above are conceptually equivalent to terms from the SNOMED-CT namespace:
 
+| Neurobagel namespace term       | External controlled vocabulary term                     |
```

Comment:
Nice. That's a really cool idea. Maybe at a later time we can add a section for these more conceptual bits of docs!